lib/elixir_internal_certificate/accounts/accounts.ex

```diff
@@ -0,0 +1,122 @@
+defmodule ElixirInternalCertificate.Accounts do
```

Comment:
The `accounts` folder name should be `account`(singular) as per our convention
![Screen Shot 2565-06-09 at 5 00 22 PM](https://user-images.githubusercontent.com/6076762/172821460-a9841a49-286f-4b01-a332-3cfadbc30a38.png)